aws-bundle/NOTICE

```diff
@@ -1,25 +1,24 @@
 
-Apache Iceberg
+Apache Iceberg (AWS Bundle)
```

Comment:
Why is this change necessary? It is clear from its inclusion in the iceberg-aws-bundle Jar.

Including this could cause confusion when this is copied and not updated, so if it isn't needed I would prefer not to add it.

Reply:
Yes, no problem to be generic and as it's in the artifact it makes sense. Doing the change.